decidim-core/app/assets/stylesheets/decidim/extras/_external-links.scss

```diff
@@ -26,3 +26,17 @@
     height: .9rem;
   }
 }
+
+.external-link-page-content{
+  .external-link{
```

Comment:
Found one more styling issue with long links:
![image](https://user-images.githubusercontent.com/864340/108503764-9474cc80-72bd-11eb-90a2-20c60d7d2801.png)

We can fix this by defining word break for the element:

```css
word-wrap: break-word;
word-break: break-word;
```

See: https://css-tricks.com/snippets/css/prevent-long-urls-from-breaking-out-of-container/

Related to #7419